packages/vite/src/node/ssr/ssrManifestPlugin.ts

```diff
@@ -96,7 +96,10 @@ export function ssrManifestPlugin(config: ResolvedConfig): Plugin {
       }
 
       this.emitFile({
-        fileName: 'ssr-manifest.json',
+        fileName:
+          typeof config.build.manifest === 'string'
+            ? config.build.manifest
```

Comment:
Should it be `config.build.ssrManifest`?

Reply:
Thanks for your review! Sorry for blind copying 😅

Reply:
Not a problem, I also approved it 😬
Thanks @remyduthu for the heads up!